src/Stats.h

```diff
@@ -17,17 +18,27 @@ class Stats
 public:
     Stats();
     void reset(StatsEnum stat);
-    void increment(StatsEnum stat, int increase = 1, float floatIncrease = 1.0);
-    void incrementFloat(StatsEnum stat, float increase = 1.0);
+    void resetAll();
+    void addDistance(int distance = 1);
+    void addTurn();
     void bindText(StatsEnum stat, QLineEdit* uiText);
-    void incrementEffectiveDistance(int distance);
-
+    void startRun();
+    void finishRun(bool reachedGoal);
+    void penalizeForReset();
```

Comment:
Can you add some comments explaining how these methods are intended to be used? For example, what is a run? When is a run considered "started" or "finished"? When should `penalizeForReset` be called? Etc.